app/controllers/audits_controller.rb

```diff
@@ -5,6 +5,7 @@ class AuditsController < ApplicationController
 
   def index
     @audits = resource_base_search_and_page.with_taxonomy_scope.preload(:user)
+    @host = resource_finder(Host.authorized(:view_hosts), params[:host_id]) if params[:host_id]
```

Comment:
please remove the back btn from the audits page 
![image](https://user-images.githubusercontent.com/20167/37873609-ca076f90-3028-11e8-8c02-4ca5f2d03474.png)